rswag-ui/package-lock.json

```diff
@@ -1,8 +1,22 @@
 {
   "name": "rswag-ui",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
These package-lock changes seem unrelated to the header changes.
Accidental inclusion?

Reply:
Thanks for the review! Yeah they appeared when I ran `./ci/build.sh` – wasn't sure if I should keep or remove so I included. Just removed though 👍🏼